pages/_app.js

```diff
@@ -1,10 +1,26 @@
+import { useState } from "react";
 import GlobalStyle from "../styles";
+import { events as eventList } from "./api/db.js";
+import EventForm from "./components/EventForm";
 
 export default function App({ Component, pageProps }) {
+  // State to store the list of events
+  const [events, setEvents] = useState(eventList);
+
+  // Function to handle form submission
+  const handleSubmit = (eventData) => {
+    setEvents([...events, eventData]);
+    setIsFormOpen([false]);
```

Comment:
There is no corresponding state for this!? It doesn't do anything. Therefore I guess you could delete this line.
```suggestion
```

Reply:
I deleted this line